### DISCUSSION TITLE
Need help with Navigation Drawer

### DISCUSSION THREAD

**Discussion by ant-subbu** [infinitered/ignite#2290] [General]
I'm working on a project using the ignite boilerplate. The navigation drawer should act like a menu to navigate to specific page (including the dynamic pages). When I used the demo code from the boilerplate code, it works like scroll menu (or web bookmarking links). But I need to do a menu which completely navigates to that particular page alone. Please help.

Thanks,

Subbu

**Comment by frankcalise** [comment]
Hi @ant-subbu -  you probably need to utilize something like [React Navigation's Drawer-based navigation](https://reactnavigation.org/docs/drawer-based-navigation/). Be sure to read the installation instructions and then follow the documented example.
